sstable/table.go

```diff
@@ -203,8 +204,15 @@ const (
 
 	pebbleDBMagic = "\xf0\x9f\xaa\xb3\xf0\x9f\xaa\xb3" // 🪳🪳
 
+	checksumLen                     = 4
+	checksummedFormatLen            = rocksDBFooterLen + checksumLen
+	checksummedFormatMagic          = "\xf0\x9f\x94\x92\xf0\x9f\x94\x92" // 🔒🔒
```

Comment:
Do we need a new magic? We could make sure the checksum is the first part of the footer, and we determine whether we read it or not based on the version (last 4 bytes of the footer before the magic).